### PR TITLE
perf: Dynamic imports of vue components

### PR DESCRIPTION
**Pull request by ArturLange**
### What

Changed components loading to dynamic. That way those can be split into chunks by `vite` and `rollup` and we can improve performance by loading smaller bundles.

Linked to #88

### CODE REVIEW

File: yarn.lock
Comment (raphodn) [inline yarn.lock:1]: this file shouldn't have changed, no ? better to do this in a seperate PR (when upgrading the dependencies)

**Review by raphodn — APPROVED** [review]
awesome !

File: yarn.lock
Comment (ArturLange) [reply yarn.lock:1]: This PR does not work with current version of Vue, I will create another to upgrade it

Comment (raphodn) [reply yarn.lock:1]: Ah really ? Because i think i already used this dynamic import mecanism elsewhere in the app

Comment (ArturLange) [reply yarn.lock:1]: On this branch it gives me the error
```
[vite:vue] Cannot read properties of undefined (reading 'start')
file: /home/alange/workspace/open-prices-frontend/src/App.vue
error during build:
TypeError: Cannot read properties of undefined (reading 'start')
    at createBlock (/home/alange/workspace/open-prices-frontend/node_modules/vue/node_modules/@vue/compiler-sfc/dist/compiler-sfc.cjs.js:1993:31)
    at /home/alange/workspace/open-prices-frontend/node_modules/vue/node_modules/@vue/compiler-sfc/dist/compiler-sfc.cjs.js:1869:55
    at Array.forEach (<anonymous>)
    at Object.parse$2 [as parse] (/home/alange/workspace/open-prices-frontend/node_modules/vue/node_modules/@vue/compiler-sfc/dist/compiler-sfc.cjs.js:1859:16)
    at createDescriptor (/home/alange/workspace/open-prices-frontend/node_modules/@vitejs/plugin-vue/dist/index.cjs:86:43)
    at transformMain (/home/alange/workspace/open-prices-frontend/node_modules/@vitejs/plugin-vue/dist/index.cjs:2302:34)
    at Object.transform (/home/alange/workspace/open-prices-frontend/node_modules/@vitejs/plugin-vue/dist/index.cjs:2848:16)
    at file:///home/alange/workspace/open-prices-frontend/node_modules/rollup/dist/es/shared/node-entry.js:25419:40
error Command failed with exit code 1.
```

Comment (raphodn) [reply yarn.lock:1]: After running which command ?

I don't have any errors showing up when running `yarn dev`

**Pull request merged**